snuba/datasets/discover.py

```diff
@@ -154,6 +154,9 @@ def __init__(self) -> None:
                 ("username", Nullable(String())),
                 ("email", Nullable(String())),
                 ("ip_address", Nullable(String())),
+                # SDK
```

Comment:
How come Geo columns don't get moved to the common list as well?

Reply:
I didn't move then because they aren't concrete columns on the transactions storage table and I thought that's what this was trying to capture.

Reply:
Let's put it on the common list. That it's found in context in transactions is just a detail of the transactions storage. (We did something similar with release, user, dist that are tags in one table)